spec/iron_bank/describe/excluded_fields/deduce_from_query_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe IronBank::Describe::ExcludedFields::DeduceFromQuery do
```

Comment:
we need test cases/context for:
- when query is nothing but valid fields
- when query is nothing but invalid fields
- when query is a mix of valid and invalid field (shuffled)

Reply:
Done